src/Client.php

```diff
@@ -143,7 +145,6 @@ class Client extends \SoapClient
         'SubscriptionPeriod'                => 'ATWS\AutotaskObjects\SubscriptionPeriod',
         'Survey'                            => 'ATWS\AutotaskObjects\Survey',
         'SurveyResults'                     => 'ATWS\AutotaskObjects\SurveyResults',
-        'ShippingType'                      => 'ATWS\AutotaskObjects\ShippingType',
```

Comment:
Intentional removal? If so, be sure to remove the object file as well.

Reply:
Hi @reynolek  Yes, intentional removal as it got declared multiple times

Reply:
@reynolek check line 142, its already there